tests/test_rss_feeds.py

```diff
@@ -28,6 +27,29 @@ def test_rss_feed(self):
         first_item = response.context["page_obj"].object_list[0]
         self.assertTrue(hasattr(first_item, "feed_url"))
 
+    def test_rss_pagination(self):
+        endpoint = reverse("rss-feed")
+        for i in range(40):
+            TestContentObj.objects.create(
+                title='Content #{}'.format(i),
+                published=timezone.now() - timezone.timedelta(days=i)
+            )
+        TestContentObj.search_objects.refresh()
+
+        client = Client()
+        resp = client.get(endpoint)
+        self.assertEqual(resp.status_code, 200)
+        page1_content_list = resp.context_data.get('content_list')
+        self.assertEqual(len(page1_content_list), 20)
```

Comment:
More thorough to check expected object ID list (vs length + "not in") checks. Could stagger publish times to get fixed ordering.